src/app/garfile/arrival/index.njk

```diff
@@ -120,56 +120,66 @@
         </div>
         {# End arrival time #}
 
-        {# Start arrival port #}
```

Comment:
Essentially took the port autocomplete and coordinate divs and placed them as containers within the radio group conditional, akin to how it works in the customs page.

This and the departure nunjucks is the same, so possibly could have templated them, if not for that being far too complicated than it was worth.